src/Internal/SortTable.elm

```diff
@@ -180,20 +195,32 @@ sortTable style model =
                         , view =
                             (case column.content of
                                 IntColumn { value, toString } ->
-                                    value >> toString
+                                    value
+                                        >> toString
```

Comment:
I moved this shared code into the code for each case, because `ElementColumn` case cannot share the code, since it's already an `Element`.

Reply:
Please rename `ElementColumn` into `CustomColumn`.